docs/manage/CLI.md

```diff
@@ -190,6 +193,107 @@ The `logout` command ends an authenticated CLI session
 viam logout
 ```
 
+### module
+
+The `module` command allows to you to manage custom modules.
+This includes:
+
+* Creating a new custom modular resource on your local filesystem
+* Updating an existing module with new changes
+* Uploading a new module to the Viam Registry
+* Updating an existing module in the Viam Registry
+
+```sh {class="command-line" data-prompt="$"}
+viam module create --organization=<org name> --name <module-id>
+viam module update --organization=<org name> --name <module-id>
+viam module upload --organization=<org name> --name <module-id> --platform <platform> --version <version> --module meta.json <packaged-module.tar.gz>
```

Comment:
Q: No `--name` for `module upload`? We infer the module name by parsing `meta.json` perhaps?